src/store/ChartTitleStore.js

```diff
@@ -51,7 +54,17 @@ export default class ChartTitleStore {
     @computed get decimalPlaces() { return this.mainStore.chart.currentActiveSymbol.decimal_places; }
     @computed get isShowChartPrice() { return this.mainStore.chart.isChartAvailable; }
     @computed get tradingTimes() { return this.mainStore.chart.tradingTimes; }
-    @computed get symbolOpenTime() { return this.tradingTimes._tradingTimesMap[this.currentSymbol.symbol].symbolOpenTime; }
+    @computed get symbolOpenTime() {
+        const times = this.tradingTimes._tradingTimesMap[this.currentSymbol.symbol].times;
+        const now = this.serverTime.getLocalDate().getTime();
+        let openTime = times ? times.find(time => time.open.getTime() > now) : null;
+
+        if (!isObjectOfTypeDate(openTime)) {
```

Comment:
```suggestion
        if (!(openTime instanceof Date)) {
```

Reply:
I thought this is going to filter invalid dates unlike instance of date, but I tested it now and it didn't do that :woman_facepalming: